1_fetch/src/get_nwis_data.R

```diff
@@ -24,12 +24,13 @@ nwis_site_info <- function(fileout, site_data){
 }
 
 
-download_nwis_site_data <- function(filepath, parameterCd = '00010', startDate="2014-05-01", endDate="2015-05-01"){
+download_nwis_site_data <- function(site_num, parameterCd = '00010', startDate="2014-05-01", endDate="2015-05-01"){
```

Comment:
According to best practices, we'd typically pull these 3 default arguments out and specify them in the `_targets.R` file, both so that anyone using the pipeline can see immediately for what parameters and dates we are pulling data, but also so that they can be easily changed without going into the function itself. Could you make that change?